### PR TITLE
Tells deploy to skip backup trough SKIP_BACKUP env var

### PR DESCRIPTION
**Pull request by rafaelp**
Sometimes I know what I'am doing and I don't want to to a full database backup on deploy. This happens when I change something small on a view, for example.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
+.DS_Store
```
Comment (maurogeorge) [inline .gitignore]: We don't need this on the project, this is specific of your machine. you need to add all system, editor or any other config to your [global gitignore](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore)

Comment (rafaelp) [inline .gitignore]: Nice! Now I do another commit removing it?

Comment (maurogeorge) [inline .gitignore]: Yeah! Please =D

**Comment by rafaelp** [comment]
@maurogeorge done.

**Comment by maurogeorge** [comment]
@marcioj @tomasmuller looks good to me. Could you guys review and merge plz.

File: lib/jumpup/heroku/integrate.rb [hunk 2]
```diff
@@ -106,6 +106,10 @@ def skip_spec?
           ENV['SKIP_SPEC'] == '1'
         end
 
+        def skip_backup?
+          ENV['SKIP_BACKUP']
```
Comment (marcioj) [inline lib/jumpup/heroku/integrate.rb]: Can we use `ENV['SKIP_BACKUP'] == '1'`? So we keep the consistency with `ENV['SKIP_SPEC'] == '1'`

Comment (rafaelp) [inline lib/jumpup/heroku/integrate.rb]: I think this is shit. I prefer to change SKIP_SPEC to accept any value.

File: spec/jumpup/heroku/integrate_spec.rb [hunk 3]
```diff
@@ -72,5 +72,13 @@
         subject.deploy_to_production
       end
     end
+    context 'skip backup' do
+      it 'does not call "backup"' do
+        ENV['SKIP_BACKUP'] = '1'
```
Comment (marcioj) [inline spec/jumpup/heroku/integrate_spec.rb]: Could you teardown these ENV's in the topmost `after` hook? Something like this:

```
after do
  ENV.delete('SKIP_BACKUP')
  ENV.delete('SKIP_SPEC')
end
```

File: lib/jumpup/heroku/integrate.rb (hunk 2 above)
Comment (marcioj) [inline lib/jumpup/heroku/integrate.rb]: I agree, I just want to both be consistence, can you update `skip_spec?` to just return `ENV['SKIP_SPEC']`?

File: spec/jumpup/heroku/integrate_spec.rb (hunk 3 above)
Comment (rafaelp) [inline spec/jumpup/heroku/integrate_spec.rb]: I just follow the same spec already exists.
I will improve everything so, but this is a matter of another pull request ;-)

Comment (rafaelp) [inline spec/jumpup/heroku/integrate_spec.rb]: @marcioj Done.

**Comment by eduardofiorezi** [comment]
:+1:
